code/modules/spells/spell_types/ethereal_jaunt.dm

```diff
@@ -71,7 +73,11 @@
 	animation.icon_state = "liquify"
 	flick("liquify",animation)
 
-
+/obj/effect/proc_holder/spell/targeted/inflict_handler/mute

```

Comment:
This is a proc you defined on the spell procholder, but on line 34 you are trying to call a mob/living/carbon proc called mute.

You probably want
 
     mute(wiz)
 
And to update the mute command to take a variable

    /obj/effect/proc_holder/spell/targeted/inflict_handler/mute(/mob/living/carbon/wiz)
       wiz.mutations | = MUT_MUTATE


You also need to handle the case of unmuting after the sleep